[dcl.enum] Enumerators don't have "initializers" (#5869)
cplusplus/draft: source/declarations.tex

An \grammarterm{enumerator-definition} with \tcode{=} gives the associated
\grammarterm{enumerator} the value indicated by the
\grammarterm{constant-expression}.
An \grammarterm{enumerator-definition} without \tcode{=} gives the associated

> \grammarterm{enumerator} the value obtained by
> increasing the value of the previous \grammarterm{enumerator} by one.
> If the first \grammarterm{enumerator-definition} has no \tcode{=}, the associated
> \grammarterm{enumerator} is given the value zero.

Reviewer: The last sentence doesn't sound like an exception to the first. It also reads better when the zero case is upfront.
```suggestion
\grammarterm{enumerator} the value zero
if it is the first \grammarterm{enumerator-definition},
and the value of the previous \grammarterm{enumerator}
increased by one otherwise.
```

Author: FWIW, I like that suggestion.

Reviewer 2: @Quuxplusone Let's apply it!

An \grammarterm{enumerator-definition} with \tcode{=} gives the associated
\grammarterm{enumerator} the value indicated by the
\grammarterm{constant-expression}.

> An \grammarterm{enumerator-definition} without \tcode{=} gives the associated

Reviewer 3: Can we quote the entire grammar snippet here, e.g. "An enumerator-definition without = _constant-expression_ ..."?

Otherwise, a twisted soul might try to circumvent this by putting "a=b" inside the constant-expression, given a suitably overloaded operator=.

Reviewer 3: Uh, I just noticed the "with =" in the preceding sentence, so it seems "without =" is fine as-is.
